lab-jeff/gulpfile.js

```diff
@@ -0,0 +1,15 @@
+const gulp = require('gulp');
+const mocha = require('gulp-mocha');
+const eslint = require('gulp-eslint');
+
+gulp.task('lint', () => {
+  return gulp.src(['**/*.js', '!node_modules/**'])
+  .pipe(eslint())
+  .pipe(eslint.format())
+  .pipe(eslint.failAfterError());
+});
+
+gulp.task('default', ['lint'], () => {
+  return gulp.src('./test/test.js')
+  .pipe(mocha({reporter: 'nyan'}));
+});
```

Comment:
Looks good, but you may want to define a separate 'test' task and then bring it into your default gulp task like you did with your lint task. That way you can run the two tasks independently if you need to.